From 0f54ed3eaec58f1eb5c9612d86a82d6744cb6fda Mon Sep 17 00:00:00 2001
From: Yi Zhao <yi.zhao@windriver.com>
Date: Thu, 23 May 2019 15:52:17 +0800
Subject: [PATCH] policy/modules/services/cron: allow crond_t to search
 logwatch_cache_t

Fixes:
avc:  denied  { search } for  pid=234 comm="crond" name="logcheck"
dev="vda" ino=29080 scontext=system_u:system_r:crond_t:s0-s15:c0.c1023
tcontext=system_u:object_r:logwatch_cache_t:s0 tclass=dir permissive=0

Upstream-Status: Pending

Signed-off-by: Yi Zhao <yi.zhao@windriver.com>
---
 policy/modules/services/cron.te | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/policy/modules/services/cron.te b/policy/modules/services/cron.te
index 474398322..5d2bc4681 100644
--- a/policy/modules/services/cron.te
+++ b/policy/modules/services/cron.te
@@ -318,6 +318,8 @@ miscfiles_read_localization(crond_t)
 
 userdom_list_user_home_dirs(crond_t)
 
+logwatch_search_cache_dir(crond_t)
+
 tunable_policy(`cron_userdomain_transition',`
 	dontaudit crond_t cronjob_t:process transition;
 	dontaudit crond_t cronjob_t:fd use;
-- 
2.17.1

